src/index.js

```diff
@@ -4,31 +4,12 @@ import enter from './assets/enter.png';
 import dots from './assets/three.png';
 import recycle from './assets/recycle.png';
 import updateItemsStatus from './js/updateItems';
-import { getListItemdata } from './js/localStorageFunctions';
+import addItems from './js/addItems';
+import clearAllCompletedtasks from './clearAllCompleted';
+import editItem from './js/editAndRemoveItem';
+import trash from './assets/trash.png'
```

Comment:
- [ ] Make sure you have a `trash.png` image in the assets folder. Also, make sure it is visible in right of the particular item. Currently it is not visible as well. Please fix it.

Reply:
sorry I didn't pay attention it's there right now